### ISSUE TITLE
Installation Honeycomb

### ISSUE THREAD

**Issue by beecm** [marsbard/puppet-alfresco#118]
Going on day three trying to set Honeycomb up in a 64bit Ubuntu 16.04LTS VMware image. Installed Oracle-java8-installer, git, and puppet. Have followed instructions on /marsbard/puppet-alfresco/ as well as tried the install process of http://fcorti.com/2015/07/20/alfresco-honeycomb-edition-how-to-run/ --- neither of which work. I am working with a clean Ubuntu image, what step am I possibly missing? The ./install process runs but has a lot of red errors - see attached. 

Need to get this concept installed and proven to get my company to fund my BeeCon2017 trip!
[install-error.txt](https://github.com/marsbard/puppet-alfresco/files/758807/install-error.txt)

**Comment by marsbard** [comment]
We had such trouble with the change in java versions between 14.04 and later versions. TBH I'm not sure why it's trying to install openjdk 7 unless you changed the java version here: https://github.com/marsbard/puppet-alfresco/blob/08150909bd755bb2bf62f1886d916ecd5010ac9d/manifests/install/jdk.pp#L3 - try making sure the version is '8' and see how you get on. Looks like fonts-droid has been removed too, try using fonts-droid-fallback here: https://github.com/marsbard/puppet-alfresco/blob/08150909bd755bb2bf62f1886d916ecd5010ac9d/manifests/packages.pp#L65

**Comment by beecm** [comment]
Both of those files were exactly as expected. Does MySql have to be installed to localhost first or is that not loaded as a part of the install? No idea why nothing is getting deployed into /opt/alfresco

**Comment by marsbard** [comment]
Nothing's going to `/opt/alfresco` because it hasn't got past the java dependency. I think I've fixed that now in commit 0e3ad1e381687ba8d683f781a29a7662daaff31f, seems to be building OK for me, can you try it and see if you get any further?

**Comment by marsbard** [comment]
In the end (my test rig is very slow right now for reasons I won't elaborate) I got the following error

```
Error: Execution of '/usr/bin/apt-get -q -y -o DPkg::Options::=--force-confold install libt1-dev' returned 100: Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package libt1-dev
Error: /Stage[main]/Alfresco::Install/Package[libt1-dev]/ensure: change from purged to present failed: Execution of '/usr/bin/apt-get -q -y -o DPkg::Options::=--force-confold install libt1-dev' returned 100: Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package libt1-dev
```

In the morning I'll see what replaces libt1-dev. If memory serves I think we can just do without it.

**Comment by marsbard** [comment]
Did a fair bit on this yesterday and today; it still doesn't start >.< but it gets a lot further.

Currently it's bombing out with a lack of a service file for tomcat; I'll try to look at that tomorrow if I can get a chance. If you can work out how to make the service file and pull request it in the `files/` folder I can make sure it gets where it needs to in puppet. The tomcat it needs to run is in `/opt/alfresco/tomcat/`

**Comment by ghost** [comment]
Attempting this install, all looks better already; will owe you a few beers in Spain!

Thanks,

Mark Goodnight, PMP, ERMp

**Comment by marsbard** [comment]
You don't owe me anything, thanks for helping prevent bitrot ;-)




---- On Fri, 10 Feb 2017 20:25:06 +0100 Mark Goodnight &lt;notifications@github.com&gt; wrote ----




Attempting this install, all looks better already; will owe you a few beers in Spain!

 

 Thanks,

 

 Mark Goodnight, PMP, ERMp

 

 Sent from my iPhone

 

 &gt; On Feb 10, 2017, at 12:44, Martin Cosgrave &lt;notifications@github.com&gt; wrote:

 &gt; 

 &gt; Closed #118 via 4cee588.

 &gt; 

 &gt; —

 &gt; You are receiving this because you authored the thread.

 &gt; Reply to this email directly, view it on GitHub, or mute the thread.

 &gt;

**Comment by ghost** [comment]
Martin,

 

I will look deeper into this, everything works as advertised except the Ubuntu menu disappeared in my test environment. I would deploy via server only for performance but odd that the Ubuntu interface would be affected by this install??

 

 

Mark
